src/themes/builder.scss

```
@@ -0,0 +1,69 @@
+    gap: 4px;
+    padding: 7px;
+    border: 1px solid var(--gooey-builder-border);
+    border-radius: 20px;
+  }
+
+  // min-height comes from `inputMinHeight` in THEME_CAPABILITIES, applied as an
+  // inline style by GooeyTextArea — not duplicated here.
+  .gooeyChat-chat-input-textarea {
+    padding: 10px 4px !important;
+    border: 0;
+    // Suppresses the base `:focus` outline; the bar itself carries the border.
```

Stylelint (17.14.0)

[error] Expected empty line before comment (scss/double-slash-comment-empty-line-before)

(scss/double-slash-comment-empty-line-before)